src/components/Todos/TodoItem.js

```diff
@@ -0,0 +1,48 @@
+import classes from './TodoItem.module.css';
+import { Link } from 'react-router-dom';
+import { convertToDate, convertToTime, calcDate } from '../../helpers/datetime';
+import { useContext } from 'react';
+import { TodoContext } from '../../store/todo-context';
+
+
+const TodoItem = ({ todo }) => {
+  const todoCtx = useContext(TodoContext);
+
+  const toggleTodoHandler = () => {
+    todoCtx.toggleTodo(todo.id, !todo.is_complete);
+  }
+
+  const completeField = todo.is_complete === true ? <p className={classes.textComplete}>Complete</p> : <p className={classes.incomplete}>Incomplete</p>
+
+  const buttonText = todo.is_complete === true ? 'Not completed' : 'Complete todo';
+
+  const calculatedDate = calcDate(new Date(todo.due_date));
+  let todoClass;
+  if (calculatedDate === 1) {
+    todoClass = `${classes.todo} ${classes.future}`;
+  } else if (calculatedDate === -1) {
+    todoClass = `${classes.todo} ${classes.past}`;
+  } else {
+    todoClass = `${classes.todo} ${classes.today}`;
+  }
```

Comment:
Tip: pro nějaké složitější manipulace s classama, než tohle, by se ti mohlo hodit [classnames](https://www.npmjs.com/package/classnames).